### ISSUE TITLE
index out of range

### ISSUE THREAD

**Issue by Chtt-123** [JaminFong/DenseNAS#10]
Hi, sorry to bother you,i just have a question:
how to deal with the problem about list index out of range in the  second part of search and in MobileNetV2 search part?

**Comment by JaminFong** [comment]
Hi, do you run the script with the provided config, and could you please post the detailed log?

**Comment by Chtt-123** [comment]
yes,i have set 10 class of img,each class has about 90 pictures.
![code1](https://user-images.githubusercontent.com/59109158/86502592-8417ec00-bdd7-11ea-8c16-705989a4cd88.png)
when i use the script "python -m run_apis.latency_measure"or "python -m run_apis.search --data_path 'the path of the split dataset'" --config configs/imagenet_search_cfg_mbv2.yaml" it turns out:
![conde2](https://user-images.githubusercontent.com/59109158/86502797-c17d7900-bdd9-11ea-9bca-30ef2c484ab5.png)
![code3](https://user-images.githubusercontent.com/59109158/86502801-c6dac380-bdd9-11ea-8145-4f6705c8afa5.png)
![code4](https://user-images.githubusercontent.com/59109158/86502803-cb06e100-bdd9-11ea-9f50-779994f6a570.png)

**Comment by JaminFong** [comment]
1. Please pass the required args when running `latency_measure`. I have specified it in the readme.
2. Do you run the script on Windows? Some of the code relies on the Linux shell command. Please run it on Linux. In addition, if you use the customized dataset with a different number of image classes, please modify the corresponding variables as I set the default values as 100 for search and 1000 for final training.
https://github.com/JaminFong/DenseNAS/blob/e479206f9d73808d4167c669bb1d045a73b6f470/models/search_space_base.py#L91
https://github.com/JaminFong/DenseNAS/blob/e479206f9d73808d4167c669bb1d045a73b6f470/models/model_derived.py#L53
Hoping this could help you!

**Comment by Chtt-123** [comment]
Ok,i will try it ,and thanks for your help again

**Comment by Chtt-123** [comment]
### How to get a higher to1_acc on MobileNetv2?
Hi,sorry to bother you again! why do i get a lower to1_acc between 10%-12% on MobileNetv2 while resnet has a higher top1_acc, i just have 10 class of images,each class has 100 pics. 
when i change the amounts  or classes numbers on images,but it has a similar situation.

**Comment by JaminFong** [comment]
Hi, do you mean the acc of the final searched model or the acc during the search? You can first verify the accuracy of a plain MobileNetV2 or ResNet on your customized dataset, then use the verified training settings to train the searched model.

**Comment by Chtt-123** [comment]
Thanks for your reply! it is the acc of final searched model, ResNet of final searched has about 80 acc on my dataset,but MobileNetV2 has 10 acc of the final searched model.

**Comment by JaminFong** [comment]
Could you please try to first train the plain mobilenetv2 on your dataset and check if the training settings are ok? If a plain not searched MBV2 achieves bad results as well, the training settings/hyper-params may need to be adjusted.
